tendrl/commons/flows/setup_ssh/__init__.py

```diff
@@ -26,5 +28,5 @@ def run(self):
         os.remove(_temp_file.name)
 
     def load_definition(self):
```

Comment:
You cannot override load_definition(), either mark this flow with class variable "internal=True" or define this flow correctly in definitions (.yml) file